README.md

```diff
@@ -1,5 +1,26 @@
 # Patra Model Card toolkit
 
+## About The Project
+
+The **Patra Model Card toolkit** is designed to assist in creating model cards that capture essential metadata for machine learning models, including fairness, explainability, and performance metrics. This toolkit simplifies the process of defining, organizing, and displaying this information, with the capability to store it in a graph database for enhanced transparency and traceability.
```

Comment:
I don't think this is quite accurate. Discuss this with @nee1k as well. it allows semi-automatic capture of metadata. Refer to the paper on this.